### DISCUSSION TITLE
Cannot find module './uws_linux_x64_79.node'

### DISCUSSION THREAD

**Discussion by rodrigooler** [uNetworking/uWebSockets.js#227] [Q&A] [answered]
The following problem occurred while upgrading Node to v13.0.1

![Captura de tela de 2019-11-15 17-53-54](https://user-images.githubusercontent.com/5496931/68975069-0dd76280-07d1-11ea-9fdd-cb731d914930.png)

**Comment by ghost** [comment, marked as answer]
Update uWS.js, you can see what happens under GitHub Releases.
